Dimit test02

Dimitrilhostis opened the pull request.

Dimitrilhostis committed:
Update test.py

Dimitrilhostis committed:
Update test.py

Excuse moi papounet

elhostis commented:
C'est bien, ça fait le job.
Essaie maintenant de créer une classe nommée guesser pour faire la même chose.
La bonne pratiue est de toujours utiliser des objets.

```
class Guesser
  def __init(self, min, max):
    self.min=min
    self.max=max
  def guess(self, v):
    ...

g = Guesser(0, 100)
g.guess
```

Dimitrilhostis committed:
Udpate

J'ai essayé de faire une classe, c'est pas tout à fait ça mais voila je te laisse regarder

elhostis commented:
Je ne veux plus voir de fonction hors de la classe.

C'est à dire qu'il faut ta classe Guesser.
Puis en dessous, qq chose de ce style

`
g = Guesser(0,10)
g.Play() 
`
